templates/aurora_postgres-master.template.yaml

```diff
@@ -183,6 +186,14 @@ Parameters:
     Default: "35"
     Description: "The number of days for which automatic database snapshots are retained."
     Type: String
+  DBEngineVersion:
+    Description: Select Database Engine Version
+    Type: String
+    Default: 9.6.9
+    AllowedValues:
```

Comment:
Is there a reason 9.6.11 is not available?

Reply:
Will add it